docs/tutorials/launch-jupyter-lab-tutorial.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 2
+---
+
+# Jupyter Lab Notebook Tutorial
+
+
+In this tutorial we will show how to open Jupyter Lab notebook
+on your computer using executor rented from fair. This way
+you can leverage your GPU to run inference and avoid paying for expensive
+cloud compute.
+
+## Pulling the Jupyter Docker Image
+
+Once you have rented a gpu and connected to the fair server, pull 
+the Jupyter Docker image. This image includes a Jupyter Notebook 
+server and various pre-installed data science packages. 
+Run the following command:
+
+```bash
+docker pull jupyter/datascience-notebook
+```
+
+This command downloads the Jupyter Docker image to your server. Depending
+on your internet connection, this may take a few minutes.
+
+## Running the Jupyter Docker Container
+
+After pulling the image, run the Docker container to start the Jupyter
+Notebook server. Use the following command:
+
+```bash
+fair docker run -p 8888:8888 jupyter/datascience-notebook
```

Comment:
I think people would prefer a PyTorch notebook by default - https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html#jupyter-pytorch-notebook

We can mention that there are other variants and point them to this page.